src/pagination/pagination.ts

```diff
@@ -7,41 +7,46 @@ import {getValueInRange, toInteger, toBoolean} from '../util/util';
 @Component({
   selector: 'ngb-pagination',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  styles: [`
+    button.page-link {
+      outline: 0;
```

Comment:
I hear you, but without outline, an user tabbing through the page won't ever notice those buttons.